tests/popup-test.js

```diff
@@ -13,10 +13,71 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+require("../readiness-tool/popup");
 
-describe('test', function() {
+var chai = require('chai');
 
-   it('should pass', function() {
-     expect(1).to.equal(1);
-   });
- });
+var vendors = require("../readiness-tool/vendors");
+
+describe('isSupported(key)', function () {
+
+  it('should return true when the key given is in the supported list', function () {
+    expect(self.popups.isSupported('Adhese')).to.be.true;
+  });
+
+  it('should return false when the key given is in NOT in the supported list', function () {
+    expect(self.popups.isSupported('alannalytics')).to.be.false;
+  });
+});
+
+describe('addToDict(tempScript, htmlString, foundThis, key, category)', function () {
+
+  var htmlString;
+  var foundThis;
+
+  beforeEach(() => {
+    htmlString = "candycanes";
+    foundThis = {
+      'supported': {
+        'ads': [],
+        'analytics': []
+      },
+      'notSupported': {
+        'ads': [],
+        'analytics': []
+      }
+    };
+  });
+
+  it('should push analytics to the analytics array', function () {
+    var tempScript = "candy";
+    var category = "Analytics";
+    var key = "comScore";
+    self.popups.addToDict(tempScript, htmlString, foundThis, key, category);
+    expect(foundThis.supported.analytics).to.include('comScore');
+    var tempScript = "peppermint";
+    key = "Swoop"
```

Comment:
It checks that regular expressions that haven't actually be found aren't being erroneously pushed to the object.